src/tools/status_tool.cpp

```diff
@@ -727,23 +763,28 @@ int StatusTool::PrintChunkserverStatus(bool checkLeftSize) {
             }
             uint64_t size = chunkNum * FLAGS_chunkSize;
             chunkLeftSize.emplace_back(size / mds::kGB);
+
             // walfilepool left size
-            metricName = GetCSLeftWalSegmentName(csAddr);
-            uint64_t walSegmentNum;
-            res = metricClient_->GetMetricUint(csAddr, metricName,
-                                                    &walSegmentNum);
-            if (res != MetricRet::kOK) {
-                std::cout << "Get left wal segment size of chunkserver "
-                          << csAddr << " fail!" << std::endl;
-                ret = -1;
-                continue;
+            if (useWalPool) {
```

Comment:
It also does not expose WAL pool metric when "useChunkFilePoolAsWalPool" .

Reply:
> It also does not expose WAL pool metric when "useChunkFilePoolAsWalPool" .

done